Readme.md

```diff
@@ -28,7 +28,8 @@ You will be asked to add the following information:
 
 * `Yoti App ID` is the unique identifier of your specific application.
 * `Yoti Scenario ID` is used to render the inline QR code.
-* `Yoti SDK ID` is the SDK identifier generated by Yoti Dashboard in the Key tab when you create your app. Note this is not your Application Identifier, which is needed by your client-side code.
+* `Yoti Client SDK ID` is the SDK identifier generated by Yoti Dashboard in the
+   Key tab when you create your app. Note this is not your Application ID.
```

Comment:
Same with this one, don't think we want the line break in the root README 👍

Reply:
Fixed in 29035a3